experimental/dds/tree2/src/feature-libraries/optional-field/optionalField.ts

```diff
@@ -450,24 +437,8 @@ export const optionalChangeRebaser: FieldChangeRebaser<OptionalChangeset> = {
 		overTagged: TaggedChange<OptionalChangeset>,
 		rebaseChild: NodeChangeRebaser,
 	) => {
-		const amended = { ...change };
-		if (change.childChanges !== undefined) {
-			const overChildChanges = new ChildChangeMap<NodeChangeset>();
-			for (const [id, overChange] of overTagged?.change.childChanges ?? []) {
-				overChildChanges.set(id, overChange);
-			}
-
-			const childChanges: typeof change.childChanges = [];
-			for (const [id, childChange] of change.childChanges) {
-				const rebasedChange = rebaseChild(childChange, overChildChanges.get(id));
-				if (rebasedChange !== undefined) {
-					childChanges.push([id, rebasedChange]);
-				}
-			}
-
-			amended.childChanges = childChanges;
-		}
-		return amended;
+		// TODO
```

Comment:
@yann-achard-MS -- meant to ask at some point, but what are the requirements on `amendRebase`? I have a vague idea it's to support cross-field moves of content, which makes me think maybe this implementation is ok. not sure though :)

Reply:
nvm, I see it's superceded by `prune` on latest main anyway.

Reply:
would like a close pair of eyes on the prune implementation. One top-level fuzz test failed on latest, but it seems likely unrelated to optional field based on failure JSON. If there isn't work filed for that, I can file it now.

Reply:
prune implementation looks good :)